### ISSUE TITLE
Any news on the Graphics mod?

### ISSUE THREAD

**Issue by enumag** [07th-mod/tatarigoroshi#7]
I'm still waiting for it. ;-)

**Comment by Maelstrom1** [comment]
As am I, but it will take some time. I am waiting for it to be stable enough for a first time playthrough.

**Comment by ItaloKnox** [comment]
The guy who used to work on it (GuraMan) disappeared. I'm giving the last touches on Umineko and I'll come back to make at least a simple graphics patch (without CGs and weather-based sprites). After I get that done we can start working through the CGs and then the weather/time sprites.

**Comment by enumag** [comment]
Damn that's what I was afraid of.

What is so difficult about CGs? Are they named differently on the console and be matched manually?

**Comment by ItaloKnox** [comment]
Actually, not much. You just need to see a video or play the original game and find where the CGs happen. Then you code them manually in the game.

**Comment by enumag** [comment]
@ItaloKnox Any news? I'm starting to worry we might not get Tatari done before Himatsu is released.

**Comment by ItaloKnox** [comment]
Yeah, not much, actually. GuraMan is still missing, I'm doing a last check on Umineko and will come back to finish this asap. Pretty bad condition we are with this patch.

**Comment by enumag** [comment]
Good to hear. It's taking a bit long but that's understandable since Umineko is much longer than a Higurashi chapter and the scripts are more complicated as well.

I read on the internet that GuraMan actually did make some proggress on this before he disappeared. Unfortunately I was unable to find his unfinished work anywhere. Did you have any luck on that front?

**Comment by ItaloKnox** [comment]
Nope, nothing. He stopped coming to the 4chan threads, so I never found a way of reaching him. Dunno what actually happened to him, but I think we can just take over and at least do a partial patch (without CGs and weather sprites/BGs). It shouldn't take too long to put a patch together, my only issue here is that I have a very slow upload speed, so I usually have to do everything through a remote machine. Shouldn't be an issue after I put together a script to automate the heavy tasks, so it's only a matter of finishing Umineko (will take a day or two) and trying to release something for Higurashi before october. I'm actually optimistic about this.

**Comment by enumag** [comment]
Nice to hear. :-)

Never mind about weather sprites/BGs. I might be able to generate those or if I'm not I'll consult with my friend who is good with PhotoShop. But what's the issue with CGs? I'd like to have those. :-P

Too bad with the upload speed. :-(

**Comment by ItaloKnox** [comment]
Well, there's no issue. The point is to just release a quick patch to add the PS3 graphics and work through the script changes (CGs and weather) after that. If we wait to release everything together, it might take way longer than two weeks to build this patch.

**Comment by enumag** [comment]
Oh you're right of course. Forgot that basic graphics patch does not require any script changes while CGs and weather do. Sorry. :-)

**Comment by enumag** [comment]
Himatsu will be out on October 7th. That's even sooner than I thought. :-)

**Comment by ItaloKnox** [comment]
Well, fuck, I need to hurry. I'll try to put everything together tomorrow and do something about Higurashi right after.

**Comment by enumag** [comment]
No need to rush too much. First there are still about 3 weeks until Himatsu. Second quality is more important than speed. Third (and most importantly) I really don't want you to overwork yourself and feel annoyed as a result. So don't worry too much about possible delays. Just do it whenever you feel like doing it and let us know if you would appriciate some help.

**Comment by ItaloKnox** [comment]
Nah, I don't think you should worry about that. Now that Umineko patch got released, I can finally focus on Higurashi. Let's see what I can finish by next week.

**Comment by DoctorDiablo** [comment]
Anything I can do to help with Higurashi?  Chapter 4 releases today and it would probably be good to wrap up ch3 before starting on 4.

**Comment by ItaloKnox** [comment]
What I need right now to put the patch together is to compare the Watanagashi graphics patch with the sprites/BGs of Tatarigoroshi. Then we compile a list of matches and see which sprites/BGs from the Watanagashi patch we won't need and which sprites/BGs from Tatarigoroshi we need to add to the patch.

Looks a little confusing but it's pretty much:
- build a list of matches between Watanagashi graphics patch and Tatarigoroshi vanilla CG/CGAlt folders
- find which sprites/BGs are used on both (always the same filename) and save them. This way we can reuse 90% of the assets
- now look for the remaining sprites/BGs from vanilla Tatarigoroshi and find their respective PS3 images [(see here for all files)](https://mega.nz/#F!SRt1DTKI!0vSWMtBLaa2VO0bDgfzb7A)
- put everything on a zip and upload

The remaining assets (only CGs, I guess) I can easily finish by integrating them to our patch, since they won't take too much space.

Unfortunately I'm having some personal issues right now and the work on the graphics patch suffered because of it. They shouldn't affect Ch.4, though, I think it's pretty much resolved right now. But as you said, Ch.4 releases today and our time is pretty short.

If you can take over the graphics patch, I can start Ch.4 already push it as much as I can before coming back to Tatarigoroshi to finish the CGs. They shouldn't take too long to finish, maybe a week at most.

**Comment by enumag** [comment]
Sounds easy enough. I think I could find some time on sunday. What about you @DoctorDiablo? Frankly it will probably take longer to download the resources than to actually do it.

**Comment by enumag** [comment]
By the way is it possible that some of the images were in Onikakushi but not in Watanagashi?

**Comment by ItaloKnox** [comment]
Maybe, it's definitely possible. Last time I started working on this, pretty much all images were covered by Watanagashi, so I didn't stop to check if Onikakushi had the remaining files.

It's actually a pretty damn easy patch to do, but my upload speeds are terrible, so I take a longer time to finish this stuff because I do everything on a remote server. If I didn't, just guess how long it would take to upload ~700mb at 60kb/s...

**Comment by enumag** [comment]
Yeah that sure sucks. :-/ Don't worry @DoctorDiablo or me can probably take care of this.

We could do the same for Himatsu too while we're at it.

Which reminds me I'll need to run the automated voice inserts for Himatsu - do we have the resources for that?

**Comment by ItaloKnox** [comment]
Many thanks for that, I can focus more on finishing the scripts while you guys are working on the graphics.

Now for Himatsu, I'm currently waiting for one of these two scenarios:
1- MangaGamer turn some affiliate earnings into store credit
or
2- the game releases on Steam

All scripts should be uploaded in at most ~6 hours. Maybe sooner if MangaGamer answers my email.

**Comment by enumag** [comment]
I was talking about the PS3 resources. I'll have my own copy of Himatsu soon.

**Comment by ItaloKnox** [comment]
Oh, my bad. Everything should be in the link I posted above. I don't know which folders are used yet, but I can check later.

**Comment by DoctorDiablo** [comment]
@enumag, I'll take a crack at it tonight.  If I can't handle it I'll let you know and you can take over.

**Comment by enumag** [comment]
@DoctorDiablo Oki, I'll look at the voice-lines insertion for Himatsu then.

**Comment by ItaloKnox** [comment]
Just using this thread to give a heads up about Himatsu: repository is now open and vanilla scripts uploaded.

Looks like I can finish everything in a week. Looks smaller than Tatari.

**Comment by enumag** [comment]
I was waiting for the steam release but their technical complications are still not resolved. Well whatever I can run the voice inserter now.

**Comment by enumag** [comment]
The script it running now. it will take a few minutes to complete. According to my info Himatsu is about half the length of the other chapters so it shouldn't be too much work.

**Comment by enumag** [comment]
https://github.com/07th-mod/himatsubushi/issues/1

**Comment by DoctorDiablo** [comment]
Ok, I finished up the Tatarigoroshi graphics patch.  Download it here: http://www.mediafire.com/file/sa9vl1y02784689/Higurashi+Ch.3+-+PS3+Graphics+Mod+1.0.rar

Also, you have to update the script to get widescreen working.  Drop this file in the Update folder: http://www.mediafire.com/file/57s97atsdxkc7vv/init.txt

It covers all the sprites, backgrounds, and effects.  There were 4 UI images I skipped because they contain text and I don't have the skills to manually recreate them in photoshop:  scenario_a, scenario_b, scenario_c, ex_otsu_tata.  We can probably do those when we add the CGs later.

**Comment by ItaloKnox** [comment]
Oh, great work! I'm going to update the repository and release a new version. :)

**Comment by enumag** [comment]
Nice work @DoctorDiablo! I've tried your patch and it's working nicely indeed. :-)

---

Now I think there are three things left.

First the PS3 version contains more character sprites than the new MG games - more faces, more clothes etc. Our patches for both Onikakushi and Watanagashi are using them. Tatari of course doesn't. I'm not quite sure how much work it would mean to do this and if it's worth it. Personally I prefer the steam sprites anyway.

Second is of course the CGs. I'm wondering if I can help with these somehow. Do we have some sort of PS3 script with japanese texts and the cgs to find where each CG belongs?

Third the sunset/night variants which GuraMan made somehow for Onikakushi and Watanagashi. Did he just copy them from PS3 or did he create them by himself? Do we want to make these too?

**Comment by ItaloKnox** [comment]
I'm not entirely sure how he got around it, since it seems our PS3 script only lists the entire text and voices, no mention of CGs or sprites. Maybe he owns the game? I have Higurashi Sui on Vita but I never really played it. Playing both side by side would take too much time, so maybe we can find a more effective way of doing it. Perhaps there's a Youtube walkthrough somewhere, so skipping through the video would be a better way of doing this.

Anyway, I think we should try to get the CGs working first, then go through the sprites and finally the weather sprites/BGs. It can take some time, but as long as we get the CGs working, we can take a longer time with the other two.

**Comment by Kamlingen** [comment]
I found the scenario images that GuraMan probly made before he went dark.

https://mega.nz/#F!BgxggTLb!w6hBR7awSRfb5jW3ohiiQQ

**Comment by DoctorDiablo** [comment]
For the sunset/night/rain variants, there are actual backgrounds in the PS3 assets that we just need to drop in and modify the script to point to.  But for the sprites it looks like we have to create orange/blue tinted variations of all the sprites used in those scenes (unless there's some way in the engine to script in a hue shift while rendering the sprites?).

We can fairly well identify the sunset/night regions by looking at when specific backgrounds are used in the script.  It also helps that the script is broken into numbered days, so each day will usually have a daytime, sunset, then night region.  Once you see bg_204, everything after that should be sunset until you hit one of the many night time bgs like bg_118, bg_121, etc.  For the rain variants, maybe we can identify those based on when the rain sound effect is playing?  But none of that applies when the story moves indoors though.

I tried finding a playthrough on youtube but could only find the steam version.  If we play the PS2 version in an emulator, it might have a fast-forward feature.  Not sure what the differences are between the PS2 and PS3 versions though.

**Comment by enumag** [comment]
I don't think the script has a built-in capability to do such thing. Or at least if it does then GuraMan didn't know about it - the patches he contributed contain the sprites with altered colors. Unfortunately I don't think he ever shared how exactly he created them.

Good note with what bgs we should look for to switch the weather. That will certainly help.

I couldn't find a youtube playthrough either. If there is a playthrough on youtube it's likely named in japanese so it would be harder to find for us. Maybe @ItaloKnox could find it?

**Comment by RukaWasTaken** [comment]
I found a playthrough of Sui on youtube, however it's all separated, from what I could gather it works like this: the uploader separated events from before/during watanagashi (maybe the watanagashi is the point when the routes get locked?), "bad ends" (not sure which arcs these fit into, opened two videos and both had Satoshi & Sonozaki Akane in them, that's all I could gather) and lastly, the events from after/during watanagashi (question arcs), Himatsubushi, the answer arcs and the arcs that were added with Sui, they are divided in two playlists (〖1/2〗 and 〖2/2〗)

I'll start listing and linking the playthroughs here:

First playlist (events from before/during watanagashi, depends on the route):

https://www.youtube.com/playlist?list=PLpZACrggG0QU5wiH0PB0l6VMxGh3vFhi5

I tried to formulate a guide to navigating this one but to no success, the only thing I could get from the title of the videos was the \* **number** *日, the 日 kanji represents "day" (iirc) so if I had to put it in a guide:

1日 - First day
2日 - Second Day
so on and so forth, but I'm not sure that would be of much help, there's also a parts later on in the playlist where the 日 stops and instead we see "route", this one gets a little bit easier since the words following the route bit are the name of the game's respective routes, so to sum it up

route レナ - Rena route (possibly leading to Onikakushi)
route 綿流し - Watanagashi route (possibly leading to-- actually no wait, it's obvious...)
route 沙都子 - Satoko route (possibly leading to Tatarigoroshi)

and now for the  (〖1/2〗) playlist:

https://www.youtube.com/playlist?list=PLpZACrggG0QXZLGvL-OacJs2NNJXdhAzh

this one's the easiest, the video titles literally have the name of the chapter the videos are a part of:

鬼隠し編 - Onikakushi-hen
綿流し編 - Watanagashi-hen
祟殺し編 - Tatarigoroshi-hen
暇潰し編 - Himatsubushi-hen
目明し編 - Meakashi-hen
罪滅し編 - Tsumihoroboshi-hen
皆殺し編 - Minagoroshi-hen
(between Minagoroshi and Matsuribayashi, there's a series of videos titled 覚めない夢, it translates to something along the lines of "I didn't wake up from the dream" or something, not sure which arc this pertains to)
祭囃し編 - Matsuribayashi-hen

there's also more (like 羞晒し編) but I haven't been able to ascertain which arc they are, same goes for the 〖2/2〗 playlist (sorry :/), anyways, here comes the second playlist.

(〖2/2〗) playlist:
https://www.youtube.com/playlist?list=PLpZACrggG0QX4INpKCfTZkfGMHTgXBLeX

Lastly, the "Bad Ends" playlist, since all I know is that it has Satoshi and Akane in some videos, I'm not sure how to form a guide for this one either:

https://www.youtube.com/playlist?list=PLpZACrggG0QU0b-UWkfKchlVmlL_kkPgg

Annnnd that's it, I hope I was able to help, even if only a little.
EDIT: Oh! forgot to mention that there's also an Omake playlist, it's this one:

https://www.youtube.com/playlist?list=PLpZACrggG0QVb6z_hxVb3mYQEs46rzKCY

**Comment by DoctorDiablo** [comment]
Nice, that's really helpful.

I tried using google translate to make navigating the videos easier... but the results were hilariously bad:
PS3 Higurashi nifty balancing a spinning washtub Hen # 7 sinister something

**Comment by Grelo** [comment]
Hi guys,
I tried to make weather sprites for Higurashi ch.3 . I wanted to look them similar to GuraGuras and i hope its close enough :D . In addition i pack them with all weather bgs which i found in PS3 assets. 
You can use them to complete the full graphic mod. If u want i can make zoom weather sprites too in future... i dont know which ones this chapter is using so i would be stuck making all of them...

[https://mega.nz/#!lYsjzBaQ!pHxvg9nvbnSUQh7leXsIUM0Qa4d4cwuYezjhywM6aCI](url)

I also looked to whether i can help with adding cgs and stuff to script but since i dont know a thing about programming scripting etc.....i tried to add them manually to update files (with logic :D ). and in that way i completed first day of tata with cgs weather etc....and it work pretty good so i continued on to second file and when i was trying to add new sprites like Tomitas and Okamuras things got pretty complicated with sprites appearing in wrong side of the screen so it was a mess..then i surrendered.....
I have access to Vita version of Higurashi even when i dont know Japanese so i was making it side by side.

If u can teach me some simple tricks with script i can do them faster but now i dont have determination to continue so i leave it to you guys....

P.S. I found some voices which match the sentence but the person who speaks is not correct, they are just short sentences nothing major....

Thanks and good luck with your work.

Enjoy :)

**Comment by enumag** [comment]
@Grelo I was trying to write program to generate the weather sprites automatically. Unfortunately I was unable to determine the right algorithm to match GuraMan's sprites so that's on hold for now. Yours look pretty well done. Thanks! :-) Can I ask how you made them?

As for how to add CGs and sprites into the script - it's hard to tell what went wrong. Maybe if you upload your work we can fix the mistakes and tell you what was wrong? It has been some time since the last patch so I sort of forgot the "tricks" - ups. But I will probably remember when I try to update something.

Voices spoken by wrong person - that's a known issue of the voice inserter I wrote. Unfortunately there is no way to for the script determine who says which line so if there are multiple voice files to match a line it just picks one. I made some optimizations to prefer one from a folder that was used by previous lines but that's about all I can do about it. The rest needs to be fixed manually so if you can tell us which lines were wrong it would be appriciated! EDIT: If you feel like doing so please open a new issue.

**Comment by Grelo** [comment]
@enumag I made the sprites in Photoshop playing with saturation, lightness, exposure, temperature etc. :) 

Ok i upload the files i made but dont laugh im worse than amateur :D 

[tata_001.txt](https://github.com/07th-mod/tatarigoroshi/files/524571/tata_001.txt)
[tata_002.txt](https://github.com/07th-mod/tatarigoroshi/files/524572/tata_002.txt)

if u need corresponding files (cgs, bgs) to these changes let me know or use them from assets.

As for the voices unfortunately i dont remember where they are playing but maybe ill check them in the future.

**Comment by enumag** [comment]
@Grelo Good. I'll try it on saturday. I don't have much time until then.

I don't think I have the corresponding CGs available. I mean they're probably somewhere in [these](https://mega.nz/#F!SRt1DTKI!0vSWMtBLaa2VO0bDgfzb7A) zip files but I'm not sure how quickly I can find them there. Do you by any chance know where they are? If not could you upload them?

**Comment by Grelo** [comment]
Okay i tried to correct my mistakes in day 2 and now it looks pretty good too. I added weather sprites and bgs (not zoom).

When u have time u can check if its ok. I can continue with this method and do all files but with slower speed.

I uploaded my whole CGAlt so u can download it(its tata01 and 02 only) + new version of the files i put there yesterday.

[https://mega.nz/#!QMc0FT5A!D0cl05p6DaehX94QBjs5Zu5bnVMKjcInmt9gFmM2ZbI](https://mega.nz/#!QMc0FT5A!D0cl05p6DaehX94QBjs5Zu5bnVMKjcInmt9gFmM2ZbI)

[tata_001.txt](https://github.com/07th-mod/tatarigoroshi/files/526832/tata_001.txt)
[tata_002.txt](https://github.com/07th-mod/tatarigoroshi/files/526831/tata_002.txt)

Have a nice day :)

**Comment by ItaloKnox** [comment]
I'm a little ashamed to say but Overwatch has been sucking my life since the newest event started, so sorry about the radio silence about a new graphics update. Let me try to get a rundown about this.

@Grelo I see your scripts have the voices already in, did you use the latest voice patch when you did it? That would save us the trouble of merging the contents, but if you didn't nothing that a couple of diffs wouldn't solve. For a few tricks on how to add things to the novel, maybe you could try to look at Onikakushi's script. Try searching for a few CGs, the pattern shouldn't be hard to get. I think you're pretty far already, so maybe learning a few keybinds on Visual Studio Code/Sublime Text/Atom would make your productivity go faster. Personally I'm a big fan of VS Code, but Sublime and Atom are also amazingly good.

@enumag I'm pretty positive that the CGs are in `HigurashiPS3-Graphics01.zip`. I remember downloading this file before and getting annoyed because I thought it had all sprites.

Anyway, this weekend I'm going back to finish Himatsubushi (only 6 scripts left), and then draft a new release for Tatarigoroshi. Best of luck, guys.

**Comment by Grelo** [comment]
@ItaloKnox Yep im using your latest voice patch. I finished 5 scripts already so with a slow tempo i continue to the next......
When i finish it all ill post it here and u can polish it a little.

Thanks for tips.

**Comment by enumag** [comment]
@ItaloKnox Nope the CGs were not there. For now I'm using the files from @Grelo. Also I think it's a bit too early for a new Tatari release. Aside from the changes for days 1 and 2 by @Grelo which I'm going over now there is nothing new since the last release.

@Grelo I've gone over 1/3 of the first day and you've indeed done a great job! I found some lines where voices didn't match the sprites and fixed them to the best of my ability. I'll ask @ItaloKnox to check it and fix the Japanese text when I'm done with day 1. If I understand correctly you're not really familiar with git, right?

**Comment by Grelo** [comment]
@enumag Just wait 2-3 days. Ill finish whole tatari graphic mod by then and upload it here.....then u can take a look on whole chapter. 
Im done with tata_012.
And yes im not familiar with git.

**Comment by enumag** [comment]
@Grelo Don't worry, it will take me at least that much time to go through the first 2 days you've already posted. I don't have nearly as much time for this as I would like to. No experience with git does not matter, I will merge the changes for you.

**Comment by Grelo** [comment]
Ok guys i finished it today since there were not much work left....finally. I have it in FHD (CGs, BGs, Sprites).
Feel free to polish and release it.  Himatsu later maybe :D .
ENJOY !!! 

Tatarigoroshi full PS3 graphic mod + scripts (with voices too).

[https://mega.nz/#!hQ91WYoa!7zWIoJDKMi_6kqK9CqgEomQ5hZU9U4hIvK1sDBBU90w](https://mega.nz/#!hQ91WYoa!7zWIoJDKMi_6kqK9CqgEomQ5hZU9U4hIvK1sDBBU90w)

Im leaving you whats left like voice errors, bugs, typos etc....good luck guys !

**Comment by Miguel202** [comment]
@Grelo  does it have the CG Art?

**Comment by enumag** [comment]
@Grelo I've created a PR with your changes: https://github.com/07th-mod/tatarigoroshi/pull/8. There is a question for you too so can you check it out?

Anyway thank you very much for your work! If you decide to help us with Himatsu or any future episode we'll greatly appriciate it as well.

**Comment by DoctorDiablo** [comment]
I extracted the text from the script files and ran it through Microsoft Word spellcheck.  It identified about 50 actual typos and a crapload of false positives.  @enumag, I'll let you finish the script updates you've been working on, then I can add these fixes on top of them.

**Comment by mistermoomoo** [comment]
Any estimate on when CGs will be done? I'm really looking forward to playing this.

**Comment by Kamlingen** [comment]
@mistermoomoo The CG patch is done and you can download it from Grelo in this thread. If you are waiting for the polished version it will probably take quite some time before it's done.

**Comment by mistermoomoo** [comment]
@Kamlingen oh! Thank you!

**Comment by enumag** [comment]
Well it seems that my speed on polishing the patch is about 1 in-game day per real-time week so it will take very long. On the other hand @Grelo did a very good job already and from what I can tell there isn't much for me to fix anyway. Personally I recommend playing now rather than waiting.

**Comment by ItaloKnox** [comment]
That's partially my fault, I guess. I'm finishing the last three Himatsu scripts, so I didn't push a new update.

It should be already posted now, Himatsu is also probably coming out very soon.

**Comment by mistermoomoo** [comment]
You guys are great! Thank you for doing this

**Comment by DifDebik** [comment]
With this patch after the scene with found body, when Keichi and others talk about their food it shows nothing but black screen and text. Is it broken or I haven't installed it properly?

**Comment by mistermoomoo** [comment]
Just finished the first section, everything seems to be working great. The parts that were black screens in the 1.0 patch are now CGs, great work guys.

**Comment by ItaloKnox** [comment]
@D0odler 
There's a chance you didn't clear the CompiledUpdateScripts folder before running the patch. This will make the game rebuild the scripts and use the new images.

**Comment by Kamlingen** [comment]
Maybe this can be used?

![ex_otsu_wata](https://cloud.githubusercontent.com/assets/22724316/19614821/0f89205a-97f9-11e6-83a9-917773e849f6.png)

**Comment by Grelo** [comment]
Hi guys, i created graphic mod for high resolution monitors (1920x1080...etc) and those who want better looking art overall (bit colour, sharpening etc.)
Its recommended to use it with the most recent graphic patch.
The file size is pretty big but the result should be positive, at least for me it is.
If you want i can upload other chapters too. 
I consider it -Final- Enjoy :)

Tatarigoroshi:
https://mega.nz/#!tIEhSYYR!74hfVpoTPKwXEs5sIGyharq4wKjgltdgknXUr0VWku8

Edit:
Onikakushi:
https://mega.nz/#!BAdhAK4D!jMKYBVotO908GkVY9hQ5SKv95-yrkABETmXFGoaupVI

Watanagashi:
https://mega.nz/#!lVsAzJwT!N95pjWZ6ampV8pBLNqm8lT7DkojRtWhKLK1wCiDou9w

P.S. In future, i think i will be doing Himatsu Graphic Mod with this method (basic graphic mod is already complete).

**Comment by DifDebik** [comment]
@ItaloKnox Oh right. Thanks a lot.

**Comment by Kamlingen** [comment]
@Grelo The Tatarigoroshi graphic mod contains the Onikakushi files.

**Comment by Grelo** [comment]
@Kamlingen Sorry, my fault. Fixed.

**Comment by mistermoomoo** [comment]
@Grelo You're a hero

**Comment by Kamlingen** [comment]
Not sure if this is something to bother with but just wanted to report it.
tata_kuruma01-07 should be using bg_103a instead of bg_100.

**Comment by Grelo** [comment]
@Kamlingen I was aware of it.... it would require doing them manually, because of different bg but OK screw it. It was easy, they are not the same as old ones but im using them. Here you go, enjoy. :) 

https://mega.nz/#!4EsTUBBL!l2aKGnDVStXxC43i4a0LZJ2_MfLmt_auSWCaa6pU9K0

As for "New chapter unlocked" BGs you need only to rename them and place it to CGAlt folder but OK....here for all chapters except Wata cause it already have it.

https://mega.nz/#!9I92hJZa!eN6enZ-YVTJQgtrDXTJ8MzY4jyvHUtF5j9z-3xQZOWA

**Comment by Kamlingen** [comment]
@Grelo Looks great, thanks for helping out on these petty complaints.

**Comment by Grelo** [comment]
@Kamlingen Im glad, that i helped :)

**Comment by ItaloKnox** [comment]
There's already some new development worth of a new version, what should I include from this thread since @Grelo's 1gb patch (around a week ago)?

**Comment by Grelo** [comment]
@ItaloKnox I dont know if the High Res. mod is worth the size ask others for feedback :D for me it is, anyway its the same content as 1GB mod. 
Other releases you should include but, it would be better to merge them to previously mentioned mod, i can update complete whole folder but have only the high res. one.

**Comment by ItaloKnox** [comment]
Yeah, there's definitely a public for the hi-res mod. I'm going to update the readme and the wiki to reflect that.

By the way, did you bundle the whole Update folder in these patches or only the patched init?

**Comment by Grelo** [comment]
@ItaloKnox Neither, just CGAlt folder. You can bundle latest Update folder.

**Comment by Ohagi-Chan** [comment]
@Grelo
Hey, I run a Steam group called Are You Ready For Watanagashi? and one of our members pointed out your mod.

Would you mind if we add a link to it on this page?
http://steamcommunity.com/groups/AYRFWata/discussions/0/451848855000361663/

**Comment by Grelo** [comment]
@Ohagi-Chan Feel free to add it. :)

**Comment by Grelo** [comment]
@ItaloKnox Can u put these changes to the next release? 1 updated script + 2 images (mistakes).

[tata_012.txt](https://github.com/07th-mod/tatarigoroshi/files/550209/tata_012.txt)

https://mega.nz/#!0dtHFZDa!bnwHYXNESL44hdxUEOoE4DQ9B2IlcSAKitYyik15b5I

**Comment by Kamlingen** [comment]
@ItaloKnox Can you also add these that Grelo as made?
https://mega.nz/#!0dtHFZDa!bnwHYXNESL44hdxUEOoE4DQ9B2IlcSAKitYyik15b5I
https://mega.nz/#!9I92hJZa!eN6enZ-YVTJQgtrDXTJ8MzY4jyvHUtF5j9z-3xQZOWA
https://mega.nz/#!4EsTUBBL!l2aKGnDVStXxC43i4a0LZJ2_MfLmt_auSWCaa6pU9K0

**Comment by Grelo** [comment]
Yep you will need them, at least the missing images because the new update already have these changes in script and without them = black screen.

**Comment by ItaloKnox** [comment]
I forgot about that, I'll recompile the graphics patch to update it and leave the links for who already downloaded the big 1080p patch.

**Comment by DoctorDiablo** [comment]
I checked in a bunch of minor spelling/grammar fixes that I found by running the script through spellcheck.

**Comment by enumag** [comment]
Nice work @DoctorDiablo! :-)
